### DISCUSSION TITLE
Is it possible to provide a mathjax config file with marp-vscode?

### DISCUSSION THREAD

**Discussion by lewallen** [marp-team/marp#72] [Q&A] [answered]
Mathjax can be configured with a standalone config file, which can be used to provide custom macros for example. Is there any way to provide such a configuration file for mathjax used by marp in vscode? Thank you!

**Comment by yhatt** [comment, marked as answer]
No. MathJax support in Marp Core is early stage and cannot be change configuration now. Even if it was supported, we would not work to make configurable in marp-vscode because of the same reason as https://github.com/marp-team/marp/discussions/71#discussioncomment-338050.
